src/Test.UnitTests.Sarif.Driver/Test.UnitTests.Sarif.Driver.csproj

```diff
@@ -35,7 +35,7 @@
     <PackageReference Include="Microsoft.Coyote" Version="$(CoyoteVersion)" />
     <PackageReference Include="Microsoft.Coyote.Test" Version="$(CoyoteVersion)" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
-    <PackageReference Include="Moq" Version="4.20.69" />
+    <PackageReference Include="Moq" Version="4.18.4" />
```

Comment:
>Version [](http://example.com/codeflow?start=36&length=7)

down-leveled??

Reply:
your description doesn't describe why we're making this change. is there a security issue in latest moq that requires us to downlevel?

Reply:
Good catch.

Reply:
Added that into description. 

According to [https://dev.azure.com/mseng/1ES/_componentGovernance/Spmi/alert/155580?typeId=252438&pipelinesTrackingFilter=0](https://dev.azure.com/mseng/1ES/_componentGovernance/Spmi/alert/155580?typeId=252438&pipelinesTrackingFilter=0) :

"This package exfiltrates data unexpectedly during installation. Do not use this version of the component. Pin to version 4.18.4."